### PR TITLE
clean up transformers scripts

### PR DESCRIPTION
**Pull request by tianleiwu**
### Description
(1) Remove class BertOptimizationOptions that has been deprecated a long time ago
(2) Move sys path setttings to `__init__.py`, and update imports
(3) Fix bert_perf_test to run properly.
(4) Fix a onnx path in a whisper test case
(5) Fix a few typos
(6) Update comments in bert_perf_test regarding to graph inputs

### CODE REVIEW

File: onnxruntime/python/tools/transformers/bert_test_data.py [hunk 1]
```diff
@@ -584,7 +584,12 @@ def create_and_save_test_data(
 
     import onnxruntime
 
-    session = onnxruntime.InferenceSession(model)
+    providers = (
+        ["CUDAExecutionProvider", "CPUExecutionProvider"]
```
Comment (kunal-vaishnavi) [inline onnxruntime/python/tools/transformers/bert_test_data.py:588]: Should we include `TensorrtExecutionProvider` as an option for performance testing?

**Comment by kunal-vaishnavi** [comment]
Can we update the default value for `operator_export_type` in `torch_onnx_export` to match the current default value in `torch.onnx.export`?

`torch_onnx_export`:
https://github.com/microsoft/onnxruntime/blob/8fd4e60c4339007943cb0d09be1fd0fe7b845119/onnxruntime/python/tools/transformers/torch_onnx_export_helper.py#L21

[`torch.onnx.export`](https://github.com/pytorch/pytorch/blob/bbf03561a9a3456e63ab5bd041e796fe59e38f54/torch/onnx/utils.py#L199): 
```
operator_export_type: _C_onnx.OperatorExportTypes = _C_onnx.OperatorExportTypes.ONNX
```

File: onnxruntime/python/tools/transformers/bert_test_data.py (hunk 1 above)
Comment (tianleiwu) [reply onnxruntime/python/tools/transformers/bert_test_data.py:588]: The tool is for generating test data and save to file, not for perf testing.

**Comment by tianleiwu** [comment]
> Can we update the default value for `operator_export_type` in `torch_onnx_export` to match the current default value in `torch.onnx.export`?

Sure. That could be in another PR (like in your PR for exporting models).

**Review by kunal-vaishnavi — APPROVED** [review]

**Pull request merged**